hatchways/team-potato-bread
# Bonus fe/in  reviews #13

### What this PR does (required):
- Adds frontend Review page/component to integrate reviews
- Integrates API for review

### Screenshots / Videos (required):
![r1](https://user-images.githubusercontent.com/60077334/133531867-3ebbf914-e249-41d9-b9df-b4c27bb04ca1.png)
![r2](https://user-images.githubusercontent.com/60077334/133531877-53d159af-c4b1-42cb-b7b5-9db9ec787de8.png)
![r3](https://user-images.githubusercontent.com/60077334/133531893-3a2a463e-22d2-46b8-8a39-ef2e3ba34a7d.png)
Video https://user-images.githubusercontent.com/60077334/133532455-b5239f4a-f9d7-464b-9535-aa4031833130.mp4

### Any information needed to test this feature (required):
- You'll need to add a link for reviews page with valid userId for whom reviews exist like `<Link to="/reviews/61417f6a98424d2da0c9b3bd">Reviews</Link>` somewhere like `NavBar`
- Click the link
- Click the More Icon on the review card to see options

### Any issues with the current functionality (optional):
- Some types are defined as: `Type | any` because currently I'm using another api which is wip to get some data
